### PR TITLE
CE-1269 Cache less agressively and version the cache

### PR DESCRIPTION
**Pull request by adamkarminski**
See [CE-1269](https://wikia-inc.atlassian.net/browse/CE-1269)

### CODE REVIEW

**Comment by macbre** [comment]
LGTM

I'll give bonus points for introducing JS "constants" to `cache.js` module (similar to what we have in `WikiaResponse` PHP class):

```php
	const CACHE_LONG = 2592000; // 30 days
	const CACHE_STANDARD = 86400; // 24 hours
	const CACHE_SHORT = 10800; // 3 hours
```

And then use it as follows:

```js
cache.set('wikiaInYourLangRequestSent' + cacheVersion, true, cache.CACHE_STANDARD);
```

**Comment by adamkarminski** [comment]
@macbre like this? :)

File: resources/wikia/modules/cache.js [hunk 1]
```diff
@@ -13,6 +13,10 @@
 		CACHE_VALUE_PREFIX = CACHE_PREFIX + 'val_',
 		CACHE_TTL_PREFIX = CACHE_PREFIX + 'ttl_',
 		CACHE_VARY_PREFIX = CACHE_PREFIX + 'vary_',
+		// Memcache-like ttl values
+		CACHE_LONG = 2592000,
+		CACHE_STANDARD = 86400,
+		CACHE_SHORT = 10800,
```
Comment (macbre) [inline resources/wikia/modules/cache.js]: Add comments how long these TTLs are and we're done here :)

**Comment by macbre** [comment]
Wunderbar, bonus points awarded to @adamkarminski  :)

**Pull request merged**